understanding/pointer-gestures.html

```diff
@@ -55,284 +53,179 @@
                </p>
-               <p>This Success Criterion does not require all functionality to be available through
-                  pointing devices, but if it is available to pointer devices then it should not require
-                  path-based gestures. While content authors generally need to provide keyboard commands
-                  or other non-pointer mechanisms that perform actions equivalent to complex gestures
-                  (see Success Criterion 2.1.1 Keyboard), this is not sufficient to conform to this
-                  Success Criterion. That is because some users rely entirely on pointing devices, or
-                  find simple pointer inputs much easier to perform and understand than alternatives.
-                  For example, a user relying on a head-pointer would find clicking a control to be
-                  much more convenient than activating an on-screen keyboard to emulate a keyboard shortcut,
-                  and a person who has difficulty memorizing a series of keys (or gestures) may find
-                  it much easier to simply click on a labeled control. Therefore, if one or more pointer-based
-                  mechanisms are supported, then their benefits should be afforded to users through
-                  simple, single-point actions alone.
+               <p>この達成基準では、全ての機能がポインティングデバイスで利用できることを要求するものではないが、ポインタデバイスで使用できる場合は、軌跡ベースのジェスチャを要求すべきでない。一般にコンテンツ制作者は、複雑なジェスチャと同等の動作を実行するキーボードコマンド又は他の非ポインタメカニズムを提供する必要があるが (達成基準 2.1.1 キーボードを参照)、この達成基準に適合するためには十分ではない。ポインティングデバイスに完全に依存している又は単純なポインタ入力が代替手段よりもはるかに実行しやすく理解しやすいと感じる利用者がいるからである。例えば、ヘッドポインタに依存している利用者は、キーボードショートカットをエミュレートするためにスクリーンキーボードを動作させるよりも、コントロールをクリックする方がはるかに使いやすいと感じるだろうし、一連のキー (又はジェスチャ) を覚えるのが困難な人は、ラベル付きコントロールをクリックする方がはるかに簡単であると感じるだろう。したがって、一つ以上のポインタベースのメカニズムがサポートされている場合、その恩恵は、単独で単純なシングルポイントによる動作を通して利用者に提供されるべきである。
                </p>
-               <p>Single pointer operations include taps and clicks, double-taps and double-clicks,
-                  long presses, swiping, dragging, and path-based gestures. Gestures such as "pinch
-                  to zoom" or two-finger swipes are <em>multipoint</em> gestures, as they require two or more pointer inputs - in this case, two fingers
-                  on a touchscreen.
+               <p>シングルポインタの操作には、タップ及びクリック、ダブルタップ及びダブルクリック、長押し、スワイプ、ドラッグ、並びに軌跡ベースのジェスチャが含まれる。「ピンチズーム」のようなジェスチャ又は 2 本指でのスワイプは、二つ以上のポインタ入力 (この場合はタッチスクリーン上の 2 本指) を必要とするため、<em>マルチポイント</em>ジェスチャとなる。
                </p>
-               <p>An exception is made for functionality that is inherently and necessarily based on
-                  complex paths or multipoint gestures. For example, entering your signature may be
-                  inherently path-based (although acknowledging something or confirming your identity
-                  need not be).
+               <p>ただし、本質的かつ必然的に複雑な軌跡又はマルチポイントジェスチャに基づく機能により例外ができる。例えば、署名の入力は (何かを認証したり、身元を確認する必要はないが) 本質的に軌跡ベースだろう。
```

Comment:
これ誤訳っぽい感じがします。この箇所は、達成基準の「ただし、マルチポイント又は軌跡ベースのジェスチャが必要不可欠である場合は例外とする」という例外について述べたものです (ということがわかりにくい原文に問題があるが……)。

括弧内ですが、need not be は need not be inherently path-based の意味かと思います。手書き署名は本質的に軌跡ベースだが、そもそも身元確認に署名が必須とは限らないわけで、身元確認が本誌的に軌跡ベースというわけではない、という話ですね。ここはちょっと補って訳した方がいいかもしれません。

たとえばこんな感じでしょうか?

```suggestion
               <p>例外は、本質的かつ必然的に複雑な軌跡又はマルチポイントジェスチャに基づく機能のために設けられている。例えば、署名の入力は本質的に軌跡ベースだろう (とはいえ、何かを認証したり、身元を確認するための機能が本質的に軌跡ベースを必要とするわけではない) 。
```

Reply:
提案をもとにして修正しました。